src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -98,33 +98,25 @@ private WebDriver createWebDriver(TestName testName) throws IOException {
         String display = getBrowserDisplay();
         switch (browser) {
         case "firefox":
-            FirefoxProfile profile = new FirefoxProfile();
-            profile.setAlwaysLoadNoFocusLib(true);
-
-            profile.setPreference(LANGUAGE_SELECTOR, "en");
-
+            FirefoxOptions firefoxOptions = new FirefoxOptions();
+            firefoxOptions.addPreference(LANGUAGE_SELECTOR, "en");
             // Config screen with many plugins can cause FF to complain JS takes too long to complete - set longer timeout
-            profile.setPreference(DOM_MAX_SCRIPT_RUN_TIME, (int)getElasticTime().seconds(600));
-            profile.setPreference(DOM_MAX_CHROME_SCRIPT_RUN_TIME, (int)getElasticTime().seconds(600));
-
-            FirefoxBinary binary = new FirefoxBinary();
-            if (display != null) {
-                binary.setEnvironmentProperty("DISPLAY", display);
```

Comment:
Why dropping `DISPLAY` support?

Reply:
This was some older code which I had cherry-picked from #284. I'll re-instate that code and try it out.

Reply:
It looks like I was a bit premature in my statement. This code must come out because it's geckodriver (as far as I can tell) which must know about the "display". I'm currently experimenting with a `geckodriver.sh` wrapper script which will do the right thing with regards to swapping environment variables around.